andela/travel_tool_front
# #162239807 Create content loaders on requests page

#### What does this PR do?
create data placeholders to represent the expected data appearance

#### Description of Task to be completed?
- Replace all the spinners on the requests, my verifications and my approvals with content placeholder loaders 

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
`ft-requests-content-loaders-162239807`
- Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the `my requests` or `my approvals` or `my verifications` page on the sidebar under `requests`
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
- Create multiple requests so as to experience how the content loaders maps to the requests you have.

#### What are the relevant pivotal tracker stories?
[#162239807](https://www.pivotaltracker.com/story/show/162239807)

#### Screenshots (if appropriate)
![ezgif com-crop 6](https://user-images.githubusercontent.com/22371596/49090954-e3169600-f26f-11e8-8ebb-a3c68f3c4873.gif)
